cpu/samd21/include/periph_cpu.h

```diff
@@ -44,6 +44,18 @@ extern "C" {
 #define SAMD21_PM_IDLE_0        (3U)    /**< Idle 0 (stops CPU)              */
 /** @} */
 
+/**
+ * @name   SAMD21 GCLK definitions
+ * @{
+ */
+enum {
+    SAM0_GCLK_MAIN = 0,                 /**< 48 MHz main clock      */
+    SAM0_GCLK_1MHZ,                     /**< 1 MHz clock for xTimer */
+    SAM0_GCLK_32KHZ,                    /**< 32 kHz clock           */
+    SAM0_GCLK_1KHZ,                     /**< 1 kHz clock            */
```

Comment:
The enums must get hard coded values. No?

Reply:
So, `SAM0_GCLK_1KHZ` is 3 (meaning GCLK3). This breaks `cpu/sam0_common/periph/wdt.c` for SAMD21, because that still uses GCLK4.

Reply:
Good catch! Changed it to `GCLK_CLKCTRL_GEN(SAM0_GCLK_1KHZ)`

Reply:
I strongly suggest to add the value of each enum.

Reply:
I'd rather discourage the use of the raw numbers altogether.

Reply:
OK, I'm not going to complain too much because I think you've done a terrific job so far.